tests/cpp_validation_tests/test_validation.cpp

```diff
@@ -2,27 +2,70 @@
 //
 // SPDX-License-Identifier: MPL-2.0
 
+#define PGM_ENABLE_EXPERIMENTAL
+
+#include "power_grid_model_cpp.hpp"
+
 #include <power_grid_model/auxiliary/dataset.hpp>
 #include <power_grid_model/auxiliary/meta_data_gen.hpp>
 #include <power_grid_model/auxiliary/serialization/deserializer.hpp>
 #include <power_grid_model/container.hpp>
 #include <power_grid_model/main_model.hpp>
 
+#include <Eigen/Dense>
```

Comment:
Please do not include eigen in this. We don't need `eigen` for validation test.

For the asymmetric attributes, just use `std::array<double,3>`;

Reply:
Okay, will do.